docs/reference/setup/cluster_restart.asciidoc

```diff
@@ -1,21 +1,24 @@
 [[restart-upgrade]]
 === Full cluster restart upgrade
 
-Elasticsearch requires a full cluster restart when upgrading across major
-versions.  Rolling upgrades are not supported across major versions. Consult
-this <<setup-upgrade,table>> to verify that a full cluster restart is
-required.
+A full cluster restart upgrade requires that you shut all nodes in the cluster
+down, upgrade them, and restart the cluster. A full cluster restart was required
+when upgrading to major versions prior to 6.x. Elasticsearch 6.x supports
+<<rolling-upgrades, rolling upgrades>> from *Elasticsearch 5.6*. Upgrading to
+6.x from earlier versions requires a full cluster restart. See the
+<<upgrade-paths,Upgrade paths table>> to verify the type of upgrade you need
+to perform.
 
-The process to perform an upgrade with a full cluster restart is as follows:
+To perform a full cluster restart upgrade:
 
-. *Disable shard allocation*
+. *Disable shard allocation.*
 +
 --
 
-When you shut down a node, the allocation process will immediately try to
-replicate the shards that were on that node to other nodes in the cluster,
-causing a lot of wasted I/O.  This can be avoided by disabling allocation
-before shutting down a node:
+When you shut down a node, the allocation process waits for one minute
+before starting to replicate the shards on that node to other nodes
+in the cluster, causing a lot of wasted I/O. This can be avoided by disabling
```

Comment:
I'd probably add something like "you can avoid racing the clock if you instead disable allocation before shutting down the node"